components/timezone/src/ixdtf.rs

```diff
@@ -327,17 +198,17 @@ impl CustomZonedDateTime<Iso> {
     ///
     /// ```
     ///
-    /// For more information on time zone parsing, see [`CustomTimeZone::try_from_ixdtf_str`].
-    pub fn try_iso_from_ixdtf_str(ixdtf_str: &str) -> Result<Self, ParseError> {
-        Self::try_iso_from_ixdtf_utf8(ixdtf_str.as_bytes())
+    /// For more information on time zone parsing, see [`CustomTimeZone::try_from_str`].
```

Comment:
suggestion: Add an extra line to link to `CustomZonedDateTime<AnyCalendar>` docs.

```suggestion
    /// For more information on time zone parsing, see [`CustomTimeZone::try_from_str`].
    /// For more information on zoned datetime parsing, see [`CustomZonedDateTime<AnyCalendar>::try_from_str`]
```

Reply:
The docs for time zone parsing were moved from CustomTimeZone to CustomZonedDateTime since the CustomTimeZone parsing function doesn't deal with the IXDTF potentially-conflicting fields. I improved the docs